docs/guides/cloud/account-management/billing-and-usage.mdx

```diff
@@ -65,10 +65,6 @@ OSS plan.
      src="/img/guides/cloud/organizations/oss-plan-3-billing.jpg"
      alt="Upgrade Plan"
    />
-   <DocsImage
```

Comment:
If this is no longer part of the flow, can you delete the image from the directory as well?